include/flatbuffers/flexbuffers.h

```diff
@@ -151,12 +151,12 @@ inline int64_t ReadInt64(const uint8_t *data, uint8_t byte_width) {
 }
 
 inline uint64_t ReadUInt64(const uint8_t *data, uint8_t byte_width) {
-  // This is the "hottest" function (all offset lookups use this), so worth
-  // optimizing if possible.
-  // TODO: GCC apparently replaces memcpy by a rep movsb, but only if count is a
-  // constant, which here it isn't. Test if memcpy is still faster than
-  // the conditionals in ReadSizedScalar. Can also use inline asm.
-  // clang-format off
+// This is the "hottest" function (all offset lookups use this), so worth
```

Comment:
There are a few indentation changes in this diff that are probably in error?

Reply:
ping

Reply:
This is from my bad clang-format at work.